### PR TITLE
Add zstandard to dependencies, fixes #229

### LINKED ISSUE #229: Add zstandard to dependencies?
Would it make sense to add zstandard to the list of dependencies? Almost all simulations in CTA are no zstd compressed, adding this would simplify the installation by one single step.

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -76,6 +77,5 @@ def finalize_options(self):
             'eventio_cut_file = eventio.scripts.cut_eventio_file:main',
         ]
     },
-    setup_requires=['pytest-runner', 'numpy'],
```
Comment (HealthyPear) [inline setup.py:79]: pytest-runner will be just removed with this change, is it correct?

File: eventio/__init__.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from .histograms import Histograms
 
 
-__version__ = '1.5.1.post1'
+__version__ = '1.5.1.post2'
```
Comment (HealthyPear) [inline eventio/__init__.py:9]: I guess this means that there will be a new release after the merging of this PR?

File: setup.py (hunk 1 above)
Comment (maxnoe) [reply setup.py:79]: Yes, `pytest-runner` enables doing `python setup.py test`, but this is not used anymore.

File: eventio/__init__.py (hunk 2 above)
Comment (maxnoe) [reply eventio/__init__.py:9]: Yes. Eventio does not yet use setuptools scm.

**Pull request merged**